### DISCUSSION TITLE
Why isn't 'brief' used instead of 'description' for hybrid commands?

### DISCUSSION THREAD

**Discussion by Kyrela** [Rapptz/discord.py#8477] [Q&A]
For the description of a slash command, when you create one using the `hybrid_command` decorator, the `description` attribute of the command is used.
However, descriptions for slash commands are limited to 100 characters, as they're displayed in-line when searching for a command.

In that case, shouldn't be `brief` used instead of `description`?
In my opinion, it makes more sense **and** it allows not to be limited to 100 characters on the `help` command.

**Comment by Rapptz** [comment]
It's because the keyword argument names are shared. Note that `short_doc` (i.e. the first line of the docstring) is used if the description is not set.
